slack-ruby-bot-server.gemspec

```diff
@@ -21,8 +21,7 @@ Gem::Specification.new do |spec|
   spec.add_dependency 'unicorn'
   spec.add_dependency 'grape'
   spec.add_dependency 'grape-swagger'
-  spec.add_dependency 'roar', '~> 1.0.4'
-  spec.add_dependency 'grape-roar'
+  spec.add_dependency 'grape-roar', '~> 0.4.0'
```

Comment:
I think this should be `>= 0.4.0` actually. There's theoretically no reason to lock a max, this would make an upgrade to 0.5.0 prohibited.